### DISCUSSION TITLE
📖✨🎃 Take a Break & Cast Your Vote! Welcome to Education Study Breaks October edition! 🎓💬🦇👻

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#140402] [GitHub Education]
Hey everyone! 🎉

We're kicking off something new and fun – **GitHub Education Study Breaks 📖!** 🎤💬 Every so often, we’ll be posting a fun question for you to answer with emojis, because we want to get to know y’all better  🙌 ♥️

👇 **Vote by clicking on an emoji** that matches your answer, and if you’ve got a better idea or option we didn’t think of, just **drop a comment below!** 🌟

For our **first question**, we wanna know:

**What’s your biggest challenge in learning to code?**

* 👀 Debugging code
* ❤️ Understanding algorithms
* 🎉 Time management
* 🚀 Staying motivated
* 👍 Other

💬 How did you fight back and defeat your biggest nemesis, **share it in the comments!** The best ones will make it into our next community check-in and student newsletter! 🙌 ✉️

Remember, every challenge you face in coding is an opportunity to grow - each error is a step closer to success. 💻 🏆

Let the votes roll in, and let’s see what coding hurdles the community is tackling the most! 🖥️💡

**Comment by sasmithaK** [comment]
Time management is the hardest thing for me. I get overwhelmed by so many new ideas and end up doing nothing. I trying to create a system to tackle this problem.

**Reply by samus-aran** [reply]
That is a fabulous answer! I'm totally the same. I feel like a create a new system every couple of months. Currently I'm trying  to split my time 80, 20, 10, 10. 80% of the week I do my biggest project until it's completed, whilst 20% is my second biggest project so it keeps it going, then 10 and 10 is everyday things that regardless I must do so I don't neglect them. 🤞 We'll see how that goes 🙈
